internal/daemonset/daemonset.go

```diff
@@ -105,6 +105,7 @@ func (dc *daemonSetGenerator) SetDriverContainerAsDesired(ctx context.Context, d
 	standardLabels := map[string]string{
 		constants.ModuleNameLabel: mod.Name,
 		dc.kernelLabel:            kernelVersion,
+		constants.NodeRole:        "module-loader",
```

Comment:
1. Why is it called `NodeRole` if the label applies to a DS (and not to a node)?
2. Can we use consts instead of `module-loader`?
3. Can we standerize the names? `module-loader` and `device-plugin-loader` or `module` and `device-plugin`?

Reply:
@qbarrand ?

Reply:
1. Indeed, not sure `NodeRole` is appropriate as a constant name - how about `DaemonSetRole`?
2. If it is used only once, I do not see the benefit of using a constant.
3. the first container loads modules, so `module-loader` sounds appropriate to me - see also the PR from @yevgeny-shnaidman https://github.com/qbarrand/oot-operator/pull/148. The device plugin, on the other hand, does not really load anything.

Reply:
Fair enough.